src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
This isn't related to using `session_id()` is it?
We don't want to ignore errors here, as they may be significant (particular the headers already sent one). The `SessionInstance` should always be initialised before headers are sent